CircleCI Build #112 failure - java.lang.AssertionError: Row id should start from current time in milliseconds after migration (current time: 1607015506230, id: 15)

```
java.lang.AssertionError: Row id should start from current time in milliseconds after migration (current time: 1607015506230, id: 15)
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at io.eventuate.common.jdbc.tests.AbstractEventuateCommonJdbcOperationsTest.assertIdSequenceUsesCurrentTimeAsStartingValue(AbstractEventuateCommonJdbcOperationsTest.java:257)
	at io.eventuate.common.jdbc.tests.AbstractEventuateCommonJdbcOperationsTest.testGeneratedId(AbstractEventuateCommonJdbcOperationsTest.java:154)
	at io.eventuate.common.jdbc.tests.AbstractEventuateCommonJdbcOperationsTest.testGeneratedIdOfEventsTableRow(AbstractEventuateCommonJdbcOperationsTest.java:127)
	at io.eventuate.common.spring.jdbc.EventuateCommonJdbcOperationsTest.testGeneratedIdOfEventsTableRow(EventuateCommonJdbcOperationsTest.java:69)

```

After commit https://github.com/eventuate-foundation/eventuate-common/commit/a3992109d00051ef0ae00b7769bbf15222e89077 that adds verification that mysql anchor event row is created,
test failed: 
```
testGeneratedIdOfEventsTableRow - io.eventuate.common.spring.jdbc.EventuateCommonJdbcOperationsTest
java.lang.AssertionError: expected:<1> but was:<0>
```
It means that row does not exist:

https://github.com/eventuate-foundation/eventuate-common/blob/wip-db-id-gen/mysql/4.initialize-database-db-id.sql#L37-L38

I see 2 possible reasons:
1. It is not created, for example because https://github.com/eventuate-foundation/eventuate-common/blob/wip-db-id-gen/mysql/9.initialization-completed.sh does not work properly
2. It was removed by some other test

--- a/_build-and-test-all.sh
+++ b/_build-and-test-all.sh
@@ -14,8 +14,6 @@ fi
 function testJdbc() {
   ${docker}Up
 
-  sleep 30
-
   ./gradlew $* :eventuate-common-micronaut-data-jdbc:cleanTest :eventuate-common-micronaut-data-jdbc:test \
   :eventuate-common-micronaut-spring-jdbc:cleanTest :eventuate-common-micronaut-spring-jdbc:test \
   :eventuate-common-spring-jdbc:cleanTest :eventuate-common-spring-jdbc:test

--- a/eventuate-common-spring-jdbc/src/test/java/io/eventuate/common/spring/jdbc/SqlDialectIntegrationTest.java
+++ b/eventuate-common-spring-jdbc/src/test/java/io/eventuate/common/spring/jdbc/SqlDialectIntegrationTest.java
@@ -55,7 +55,6 @@ public class SqlDialectIntegrationTest {
   public void testAddLimitToSimpleSelect() {
     final int LIMIT = 3;
 
-    clearEventsTable();
     prepareRandomData();
 
     String sqlWithoutLimit = String.format("select * from %s", DEFAULT_EVENTUATE_SCHEMA.qualifyTable("events"));
@@ -64,7 +63,7 @@ public class SqlDialectIntegrationTest {
     List<Map<String, Object>> resultWithLimit = jdbcTemplate.queryForList(sqlWithLimit);
     List<Map<String, Object>> resultWithoutLimit = jdbcTemplate.queryForList(sqlWithoutLimit);
 
-    Assert.assertEquals(DEFAULT_DB_RECORDS, resultWithoutLimit.size());
+    Assert.assertTrue(resultWithoutLimit.size() >= DEFAULT_DB_RECORDS);
     Assert.assertEquals(LIMIT, resultWithLimit.size());
   }
 
@@ -73,7 +72,6 @@ public class SqlDialectIntegrationTest {
     final int LIMIT = 2;
     final int SELECTABLE_RECORDS = 3;
 
-    clearEventsTable();
     prepareRandomData();
 
     String eventType = generateId();
@@ -117,10 +115,6 @@ public class SqlDialectIntegrationTest {
 
   private void assertAllRowsHaveTheSameEventType(List<Map<String, Object>> rows, String eventType) {
     Assert.assertTrue(rows.stream().allMatch(row -> row.get("event_type").equals(eventType)));
-  }
-
-  private void clearEventsTable() {
-    jdbcTemplate.execute(String.format("truncate table %s", DEFAULT_EVENTUATE_SCHEMA.qualifyTable("events")));
   }
 
   private void prepareRandomData() {

